temporalcli/commands.taskqueue_get_build_id.go

```diff
@@ -0,0 +1,126 @@
+package temporalcli
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/fatih/color"
+	"github.com/temporalio/cli/temporalcli/internal/printer"
+	"go.temporal.io/sdk/client"
+)
+
+func (c *TemporalTaskQueueGetBuildIdsCommand) run(cctx *CommandContext, args []string) error {
+	cl, err := c.Parent.ClientOptions.dialClient(cctx)
+	if err != nil {
+		return err
+	}
+	defer cl.Close()
+
+	sets, err := cl.GetWorkerBuildIdCompatibility(cctx, &client.GetWorkerBuildIdCompatibilityOptions{
+		TaskQueue: c.TaskQueue,
+		MaxSets:   c.MaxSets,
+	})
+	if err != nil {
+		return fmt.Errorf("unable to get task queue build ids: %w", err)
+	}
+
+	type rowtype struct {
+		BuildIds      []string
+		DefaultForSet string
+		IsDefaultSet  bool
```

Comment:
You'll probably want to provide `"json:buildIds"` type of struct tags for these since we usually want our JSON as camelCase (even if old CLI wasn't as kind/consistent).

Reply:
Done